pages/dkp/kommander/2.2/projects/applications/platform-applications/index.md

```diff
@@ -50,6 +50,6 @@ To use the CLI to enable or disable applications, see [Application Deployment](.
 
 ## Upgrade Platform applications from the CLI
 
-Platform Applications within a Project are automatically upgraded when the Workspace that a Project belongs to is upgraded. See [Upgrading Workspace Platform Applications][upgrade_workspaces] for more information on how to upgrade these applications.
+Platform Applications within a Project are automatically upgraded when the Workspace that a Project belongs to is upgraded. See [Upgrade Kommander][upgrade_workspaces] for more information on how to upgrade these applications.
```

Comment:
I'm still getting a 404 not found when I click this link on Dev-docs

Reply:
alright, I suck at this. One more try.

Reply:
Fixed!!

Reply:
There it goes!!! Woohoo